spec/lib/saml_idp/name_id_formatter_spec.rb

```diff
@@ -7,7 +7,7 @@ module SamlIdp
       let(:list) { { email_address: ->() { "foo@example.com" } } }
 
       it "has a valid all" do
-        expect(subject.all).to eq(["urn:oasis:names:tc:SAML:2.0:nameid-format:emailAddress"])
+        expect(subject.all).to eq(["urn:oasis:names:tc:SAML:1.1:nameid-format:emailAddress"])
```

Comment:
Can we have some specs that use 2.0? Or show what it takes to make this behave either way?